curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/python-iteration-and-modules.english.md

```diff
@@ -17,10 +17,23 @@ videoId: XzosGWLafrY
 question:
   text: How would you iterate over and print the keys and values of a dictionary named "user"?
   answers:
-    - '<pre>for key in user.items():<br>    print(key)</pre>'
-    - '<pre>for key, value in user.all():<br>    print(key, value)<br>    print(value)'
-    - '<pre>for key, value in user.items():<br>    print(key, value)</pre>'
-    - '<pre>for key, value in user<br>    print(key, value)</pre>'
+    - ```python
+      for key in user.items():
+          print(key)
+      ```
+    - ```python
+      for key, value in user.all():
+          print(key, value)
+          print(value)
+      ```
+    - ```python
+      for key, value in user.items():
+          print(key, value)
+      ```
+    - ```python
+      for key, value in user
+          print(key, value)
+      ```
```

Comment:
```suggestion
    - |
       ```python
       for key in user.items():
           print(key)
       ```
    - |
       ```python
       for key, value in user.all():
           print(key, value)
           print(value)
       ```
    - |
       ```python
       for key, value in user.items():
           print(key, value)
       ```
    - |
       ```python
       for key, value in user
           print(key, value)
       ```
```